addons/event/static/src/views/event_slot_calendar/event_slot_calendar_renderer.js

```diff
@@ -2,9 +2,54 @@ import { CalendarCommonRenderer } from "@web/views/calendar/calendar_common/cale
 import { CalendarRenderer } from "@web/views/calendar/calendar_renderer";
 import { CalendarYearRenderer } from "@web/views/calendar/calendar_year/calendar_year_renderer";
 
+import { useEffect } from "@odoo/owl";
+
 export class EventSlotCalendarCommonRenderer extends CalendarCommonRenderer {
     // Display end time and hide title on the full calendar library event.
     static eventTemplate = "event.EventSlotCalendarCommonRenderer.event";
+    // Make sure that slots outside of the event time range (greyed-out from the useEffect)
+    // aren't selectable by the calendar multi create square selection.
+    static cellIsSelectable = (cell) => !cell.classList.contains("o_event_disabled_day");
+
+    setup() {
+        super.setup(...arguments);
+        this.rangeStartDate = this.props.model.meta.context.event_start_date;
+        this.rangeEndDate = this.props.model.meta.context.event_end_date;
+        useEffect(
+            // Add overlay to disable days outside of event time range.
+            (el) => {
+                if (!this.rangeStartDate || !this.rangeEndDate) {
+                    return;
+                }
+                el.querySelectorAll(".fc-day:not(.fc-col-header-cell)").forEach((dayEl) => {
+                    if (dayEl.dataset.date < this.rangeStartDate || dayEl.dataset.date > this.rangeEndDate) {
+                        dayEl.classList.add("bg-secondary", "bg-opacity-25", "o_event_disabled_day");
+                    }
+                });
```

Comment:
So we restrict to days and not more precisely. We always are on the month view when creating slots? all good then